### ISSUE TITLE
Missing steps in Vagrant install document

### ISSUE THREAD

**Issue by CraigSterrett** [clearlinux/cloud-native-setup#321]
There appears to be some information missing from some of the documentation on setting up Vagrant.
 
This top-level document: https://github.com/clearlinux/cloud-native-setup/tree/master/clr-k8s-examples says to either
Follow instructions in the Vagrant docs
Or, follow our detailed steps
Now you have a 3 node cluster up and running. 
The 'detailed steps' link is to this document: https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/vagrant.md  In about the middle of that document, immediately after installing the software, it says to run "vagrant up" (with or without CPU and memory settings, and with or without specifying "clr-01" as the type of VM to create).
 
The first document says that after following those steps from the second document, you'll have a cluster up and running.  What actually happens is you get this message in red:
A Vagrant environment or target machine is required to run this
command. Run `vagrant init` to create a new Vagrant environment. Or,
get an ID of a target machine from `vagrant global-status` to run
this command on. A final option is to change to a directory with a
Vagrantfile and to try again.
 
It appears the above documents are missing the "vagrant init" step described in the chained documents pointed to by the 'Vagrant docs' link:
https://www.vagrantup.com/intro/getting-started/install.html#installing-vagrant
https://www.vagrantup.com/intro/getting-started/project_setup.html
 
I would think it would be appropriate to flag as a bug in the '.../vagrant.md' document that it fails to mention the necessary "vagrant init" step.  I would think somebody from the outside who tries to follow the recipe will lose some confidence in the quality of the documentation.
 
Thanks,
 
Robert Riches

**Comment by inmanturbo** [comment]
Indeed. I stumbled across this compelling and interesting guide today, and begun following blindly; much to my dismay I was wasting my time. The plan was to put together a little prototype lab to pitch migrating a great deal of services to Clear Linux, as I happen to like the distro very much. Initially I was very pleased to find a central source for distro-centric instructions, thinking it would save me a great deal of time and gain me some insights as well. 

Unfortunately there are many holes. The cloud native docs do not anywhere lead to a meaningful example of a basic cluster. The vagrant docs are missing the steps mentioned above, and I could find no supported Vagrant Cloud image to pull when I tried to put the rest together on my own. Was this just a whim that was later abandoned by Clear Linux (?) I begin to lose faith in more than just the docs when things begin to look abandoned. Is this:
- https://clearlinux.org/blogs-news/clear-linux-os-now-certified-kubernetes-distribution

Going the same route as this:
- https://community.clearlinux.org/t/clear-linux-os-will-be-reducing-its-java-footprint/2454
- https://community.clearlinux.org/t/compiling-spark-from-source/4937
- https://docs.01.org/clearlinux/latest/tutorials/apache-spark.html#apache-spark [There is no mention here of the deprecation]
(?)

Sadly the path to a working prototype of the cloud native model on Clear Linux is no different than it would be on a custom distro that I just slapped together myself this morning: Starting with penciling pseudo-code into a Makefile. 

I don't expect to be spoon fed all the answers in the documentation, and don't mean for this to be a criticism at all. The OS is far superior to others for cloud deployments, and cloud platform development. And the devs are fantastic: responding quickly to clear up bugs and issues. Unfortunately they are also having to field a lot of "questions" posed as "bugs" and while it is nice to get a devs advice on how to deal with questions that arise for common deployments and such, it shouldn't always be necessary.

**Comment by inmanturbo** [comment]
Just saw the Vagranfile in this repo. Going backdown the rabbit hole  . . .

**Comment by inmanturbo** [comment]
If anyone comes across these errors just clone this repo and cd into cloud-native-setup/clr-k8s-examples/ before running the commands found in the vagrant.md document :facepalm: 

Works very well. Excellent example. Kata is a bit much in many actual use cases but still very good as an example despite being a little on the bulky side. Brings up a lot of infrastructure -- `all` option even brings up Rook!
